content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -41,15 +40,15 @@ Automatically close inactive tickets
 ====================================
 
 Tickets that are inactive for a set period of time can be automatically closed. At that point, they
-will be moved to a folded stage.
+are moved to a folded stage.
 
-Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
-Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
+Go to the team's settings page by going to :menuselection:`Helpdesk app --> Configuration -->
+Helpdesk Teams`. Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
 
-If one of the team's stages is set to be folded in the kanban view, it will be the default selection
-in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the stage that
-occurs first in the pipeline will be the default. If no stage is folded, the default selection will
-be the last stage in the pipeline.
+If one of the team's stages is set to be folded in the Kanban view, the folded stage is the default
+selection in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the
+stage that occurs first in the pipeline is the default. If no stage is folded, the default selection
+is the last stage in the pipeline.
 
 The :guilabel:`After days of inactivity` field defaults to `7`, but can be adjusted if necessary.
```

Comment:
```suggestion
The :guilabel:`After (#) days of inactivity` field defaults to `7`, but can be adjusted if necessary.
```

Not sure if that's the best way, but I just wanted to note that there is a value between the text up there.  Maybe "After X days"?